extensions/parquet/table/src/main/java/io/deephaven/parquet/table/ParquetInstructions.java

```diff
@@ -546,6 +594,18 @@ public boolean useDictionary(final String columnName) {
             return getOrDefault(columnName, false, ColumnInstructions::useDictionary);
         }
 
+        @Override
+        public OptionalInt getFieldId(String columnName) {
```

Comment:
Instead of implementing this method, you can update the getOrDefault method overload which accepts a string to do
```
        private <VALUE_TYPE> VALUE_TYPE getOrDefault(final String columnName, final VALUE_TYPE defaultValue,
                final Function<ColumnInstructions, VALUE_TYPE> fun) {
            if (columnNameToInstructions == null) {
                return defaultValue;
            }
            final ColumnInstructions ci = columnNameToInstructions.get(columnName);
            if (ci == null) {
                return defaultValue;
            }
            return fun.apply(ci);
        }
```

And then we should be able to use something like
```
        public String getColumnNameFromParquetFieldId(int fieldId) {
             return getOrDefault(columnName, OptionalInt.empty(), ColumnInstructions::getFieldId);
        }
```

Reply:
I don't think `getColumnNameFromParquetFieldId` can be simplified in this way, but getFieldId can.

Reply:
My bad. That's what I meant, thanks. Updated my original comment too.